piptools/scripts/compile.py

```diff
@@ -236,6 +236,20 @@ def _get_default_option(option_name: str) -> Any:
     default=True,
     help="Add options to generated file",
 )
+@click.option(
+    "--unsafe-packages",
+    multiple=True,
+    help="List of packages to consider unsafe. Replaces "
```

Comment:
I think the help text here is a bit misleading regarding the required syntax, suggesting that maybe the user ought to provide a CSV. I would like to bring it more in line with the descriptions for our other options which specify `multiple=True`.

Reply:
Done. I adjusted the help to be more similar to --upgrade-package.

Reply:
Thanks! Maybe the user-facing `--unsafe-packages` can be renamed as singular, `--unsafe-package`?

I'm not tied to the "unsafe" term, but either way I think it should be singular.

Reply:
Done. I'm neutral on name and other names like --exclude-package are fine. Only specific pro to keep unsafe-package is it replaces existing unsafe package list (default excludes) of pip/setuptools.